proxy/Main.cc

```diff
@@ -1943,17 +1943,28 @@ main(int /* argc ATS_UNUSED */, const char **argv)
       int delay_p = 0;
       REC_ReadConfigInteger(delay_p, "proxy.config.http.wait_for_cache");
 
+      // Use a condition variable to check if we are ready to call
```

Comment:
What the heck does that have to do with the following code? Need better comment here. Looks like we always wait for the `ET_NET` threads to be ready.